
[31m[2022-07-14 14:34:40,893] [Request.py:55] [Request:send_request] [ERROR]- unkown error[0m

[31m[2022-07-14 14:34:40,893] [Assert.py:53] [Assert:assert_in] [ERROR]- 预期结果类型：<class 'dict'>或实际结果类型：<class 'tuple'>格式有错误，请排查！[0m
